socorro/external/boto/connection_context.py

```diff
@@ -73,58 +72,39 @@ class ConnectionContextBase(RequiredConfig):
         reference_value_from='resource.boto'
     )
 
-    operational_exceptions = (
+    RETRYABLE_EXCEPTIONS = (
```

Comment:
Here's the new model: the connection context defines an `is_retryable_exception` method which determines yes or no as to whether it's retryable. If it's retryable, then `retry` will recycle the connection and try again. That radically simplifies the two different classes of exceptions and the middling ground that `is_operational_exception` occupied.